packages/govuk-frontend/src/govuk/components/checkboxes/checkboxes.mjs

```diff
@@ -49,12 +52,21 @@ export class Checkboxes extends GOVUKFrontendComponent {
     this.$inputs.forEach(($input) => {
       const targetId = $input.getAttribute('data-aria-controls')
 
-      // Skip checkboxes without data-aria-controls attributes, or where the
-      // target element does not exist.
-      if (!targetId || !document.getElementById(targetId)) {
+      // Skip radios without data-aria-controls attributes
+      if (!targetId) {
         return
       }
 
+      const targetElement = document.getElementById(targetId)
+
+      // Throw if target conditional element does not exist.
+      if (!targetElement) {
+        throw new ElementError(targetElement, {
```

Comment:
Sorry Brett now it's moved I've noticed we only use it when it's `null`

```suggestion
      // Throw if target conditional element does not exist.
      if (!document.getElementById(targetId)) {
        throw new ElementError(null, {
```

Reply:
Oh nice catch!